%YAML:1.0
stereo0:
  cam0:
    cam_overlaps: [1]
    camera_model: pinhole
    distortion_coeffs: [-0.3703768198187131, 0.20390389297743816, 0.003666992944255762, 0.002886184903336824, -0.06989334535810098]
    distortion_model: radtan
    intrinsics: [775.1026439020942, 691.9002200404184, 620.9221230536346, 319.44507635916455]
    resolution: [1280, 640]
    rostopic: /cam0/image_raw
  cam1:
    T_cn_cnm1:
      - [0.9999637731170729, 0.0007270257798959878, 0.00848079518575237, -0.06294175936842204]
      - [-0.0007235534334902909, 0.999999653158084, -0.00041249744293890306, 4.190098104745202e-05]
      - [-0.008481092140532278, 0.00040634619096695517, 0.9999639523297212, -0.0001839544807564846]
      - [0.0, 0.0, 0.0, 1.0]
    cam_overlaps: [0]
    camera_model: pinhole
    distortion_coeffs: [-0.3780555301451926, 0.2444940656668863, 0.0030747773827443006, 0.0015973082061836769, -0.11461206059457467]
    distortion_model: radtan
    intrinsics: [774.4644918793441, 691.6152070859955, 631.9739731653336, 308.94946291137256]
    resolution: [1280, 640]
    rostopic: /cam1/image_raw
